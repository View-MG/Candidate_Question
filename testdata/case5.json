[
  {
    "no": 1,
    "platformProductId": "FG0A-CLEAR-OPPOA3/%20xFG0A-CLEAR-OPPOA3-B/FG0A-MATTE-OPPOA3",
    "qty": 1,
    "unitPrice": 120,
    "totalPrice": 120
  }
]