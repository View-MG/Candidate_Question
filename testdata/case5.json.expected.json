[
  {
    "no": 1,
    "productId": "FG0A-CLEAR-OPPOA3",
    "materialId": "FG0A-CLEAR",
    "modelId": "OPPOA3",
    "qty": 1,
    "unitPrice": 40.00,
    "totalPrice": 40.00
  },
  {
    "no": 2,
    "productId": "FG0A-CLEAR-OPPOA3-B",
    "materialId": "FG0A-CLEAR",
    "modelId": "OPPOA3-B",
    "qty": 1,
    "unitPrice": 40.00,
    "totalPrice": 40.00
  },
  {
    "no": 3,
    "productId": "FG0A-MATTE-OPPOA3",
    "materialId": "FG0A-MATTE",
    "modelId": "OPPOA3",
    "qty": 1,
    "unitPrice": 40.00,
    "totalPrice": 40.00
  },
  {
    "no": 4,
    "productId": "WIPING-CLOTH",
    "qty": 3,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 5,
    "productId": "CLEAR-CLEANNER",
    "qty": 2,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 6,
    "productId": "MATTE-CLEANNER",
    "qty": 1,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  }
]